server/middlewares/index.js

```diff
@@ -6,7 +6,7 @@ const { User } = db;
  *Users endpoint middlewares
  *
  * @export
- * @class UsersMiddleware
+ * @class UsersMiddlewares
  */
 export default class UsersMiddleware {
```

Comment:
Please update the class name to match the new documentation

Reply:
Acknowledge

Reply:
I have fixed it